packages/d3fc-element/src/css.js

```diff
@@ -1,16 +1,22 @@
 // Adapted from https://github.com/substack/insert-css
-const css = `d3fc-canvas,d3fc-svg{position:relative;display:block}\
+export const css = `d3fc-canvas,d3fc-svg{position:relative;display:block}\
 d3fc-canvas>canvas,d3fc-svg>svg{position:absolute;height:100%;width:100%}\
 d3fc-svg>svg{overflow:visible}`;
 
-const styleElement = document.createElement('style');
-styleElement.setAttribute('type', 'text/css');
+export const insertCss = (node, css, id) => {
+    const root = node.getRootNode();
```

Comment:
I think we might have compatibility problems using this -https://developer.mozilla.org/en-US/docs/Web/API/Node/getRootNode

Reply:
I'll leave it up to your judgement.

We still can't ingest the CSS at build time, only insert it at runtime. I suspect we won't use #1226 for Perspective because importing from "@d3fc/d3fc-chart" instead of from "d3fc" causes errors in our webpack build output.